tasks/affective_task/server_affective_task.py

```diff
@@ -88,7 +92,7 @@ def run(self, images_dir: str, collaboration: bool = False):
                             "selected_rating_type": response["update"]["rating_type"],
                             "selected_rating": response["update"]["rating_index"] - 2
                         }
-                        self._csv_writer.writerow([time(), image_path, client_name, json.dumps(record_activity)])
+                        self._csv_writer.writerow([time(), monotonic_time, boot_time, image_path, client_name, json.dumps(record_activity)])
```

Comment:
We should label each column in the CSV file in the __init__ function of this class. Simply write the first row as ["time", "monotonic_time", "boot_time", "image_path", "participant_id", "activity"]